README.md

```diff
@@ -23,9 +23,21 @@ In your Jenkins project, install the plugin.
   3. Click **Choose File** and select the IBM Bluemix DevOps plugin installation file. 
   4. Click **Upload**.
   5. Restart Jenkins and verify that the plugin was installed.
-
+  
 ## 3. Integrate DevOps Insights with your Jenkins project
 
+Configure your Jenkins jobs to send notifications to your toolchain.
```

Comment:
Is this title or bullet point? Looks like it is the title/summary of this section, if so we need to change the format